web/auth/auth.go

```diff
@@ -513,6 +520,75 @@ func checkRegistrationToken(next echo.HandlerFunc) echo.HandlerFunc {
 	}
 }
 
+func passphraseResetForm(c echo.Context) error {
+	if middlewares.IsLoggedIn(c) {
+		instance := middlewares.GetInstance(c)
+		redirect := defaultRedirectDomain(instance).String()
+		return c.Redirect(http.StatusSeeOther, redirect)
+	}
+	return c.Render(http.StatusOK, "passphrase_reset.html", nil)
+}
+
+func passphraseReset(c echo.Context) error {
+	instance := middlewares.GetInstance(c)
+	// TODO: check user informations to allow the reset of the passphrase since
+	// this route is of course not protected by authentication/permission check.
+	if err := instance.RequestPassphraseReset(); err != nil {
+		return err
+	}
+	// Disconnect the user if it is logged in. The idea is that if the user
+	// (maybe by accident) asks for a passphrase reset while logged in, we log
+	// him out for him de re-go through the process of logging back-in. It is
+	// more a UX choice than a "security" one.
+	if middlewares.IsLoggedIn(c) {
+		session, err := sessions.GetSession(c, instance)
+		if err == nil {
+			c.SetCookie(session.Delete(instance))
+		}
+	}
```

Comment:
Oh shoot, I thought it was "automatic" (and activated on all routes of `/auth`) but of course I need to add the token in the template and the routes.